src/main/java/pl/javastart/task/Car.java

```diff
@@ -0,0 +1,34 @@
+package pl.javastart.task;
+
+public class Car extends Vehicle {
+    boolean isAcOn;
+
+    public Car(String name, double tankCapacity, double averagePetrolConsumption, boolean isAcOn) {
+        super(name, tankCapacity, averagePetrolConsumption);
+        this.isAcOn = isAcOn;
+    }
+
+    public boolean isAcOn() {
+        return isAcOn;
+    }
+
+    public void setAcOn(boolean acOn) {
+        isAcOn = acOn;
+    }
+
+    @Override
+    public double range() {
+        return (getTankCapacity() * 100) / computeAveragePetrolConsumption();
+    }
```

Comment:
Ogólny algorytm się nie zmienia i zasięg to zawsze "pojemność baku / aktualne spalanie * 100km". Jedyne co się zmienia, to aktualne spalanie, czyli metoda `computeAveragePetrolConsumption`